packages/client/hmi-client/src/components/widgets/tera-chart-settings-panel.vue

```diff
@@ -96,6 +96,23 @@
 								<input type="color" :value="v.primaryColor ?? ''" @change="onComparisonChange(v.name, $event)" />
 							</template>
 						</tera-chart-settings-item>
+						<Divider />
+					</section>
+					<section v-if="isChartLabelsOptionEnabled" class="items-wrapper">
+						<h5>Chart Labels</h5>
+						<div>
+							<label>Title</label>
+							<tera-input-text placeholder="No title" v-model="chartLabelTitle" />
+						</div>
+						<div>
+							<label>X axis</label>
+							<tera-input-text placeholder="No x axis" v-model="chartLabelXAxis" />
+						</div>
+						<div>
+							<label>Y axis</label>
+							<tera-input-text placeholder="No y axis" v-model="chartLabelYAxis" />
+						</div>
```

Comment:
Their is a `label` prop.

```suggestion
						<tera-input-text label="Title" placeholder="No title" v-model="chartLabelTitle" />
						<tera-input-text label="X axis" placeholder="No x axis" v-model="chartLabelXAxis" />
						<tera-input-text label="Y axis" placeholder="No y axis" v-model="chartLabelYAxis" />
```

Reply:
Done